tasks/51_abbreviated-name/abbreviated-name.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * abbreviated - Converts a fullname in abbreviated form.
+ *
+ * @example
+ * //return 'Verbovyi D. O.'
+ * abbreviated('Verbovyi Dmytro Oleksandrovych');
+ * @param {String} str - fullname that function will convert.
```

Comment:
which instead of that ?

Reply:
@vovcyan //return Verbovyi D. O. instead of //return 'Verbovyi D. O.',  i think